extensions/build.gradle

```diff
@@ -20,8 +20,6 @@ subprojects {
     sourceCompatibility = 1.7
     targetCompatibility = 1.7
 
-    compileJava.options.encoding = 'UTF-8'
```

Comment:
This line is somewhat concerning; I've hit all sorts of issues when the encoding is not explicitly set to UTF-8, typically on Windows. Recall that Java uses the platform encoding if unspecified.